core/src/main/scala/scala/macros/core/Universe.scala

```diff
@@ -0,0 +1,95 @@
+package scala.macros.core
+
+import scala.macros.internal.trees.Gensym
+
+trait Universe extends Gensym {
```

Comment:
A minor issue with the name `Gensym` -- it doesn't generate any symbols, but just unique strings.

Reply:
I agree, once DottyUniverse gets a Context then we can go ahead and replace Gensym with a TermNameFresh(prefix: String) method in the API.